naff/client/const.py

```diff
@@ -86,10 +86,16 @@
 __py_version__ = f"{_ver_info[0]}.{_ver_info[1]}"
 __api_version__ = 10
 logger_name = "naff"
-logger = logging.getLogger(logger_name)
+_logger = logging.getLogger(logger_name)
 default_locale = "english_us"
 kwarg_spam = False
 
+
+def logger() -> logging.Logger:
```

Comment:
If you make this an `@property`, you don't need to change everything to `logger()`, and it won't be a breaking change.

Reply:
Properties only work in instantiated classes tho, which this is not. Or am I missing something here?